src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react"
+import { getArticleById } from "../../utils/utils"
+import { useParams } from "react-router-dom"
+import Home from "./Home"
+
+
+function SingleArticle(){
+    const [article, setArticle] = useState()
+    const {article_id} = useParams()
+
+    useEffect(()=>{
+        getArticleById(article_id).then((response)=>{
+            console.log(response.data.article)
+            setArticle(response.data.article)
+        })
+    },[article_id])
+
```

Comment:
You could add some loading state here to conditionally render something to tell a user the data is still being fetched while they are waiting ( for those on a slower internet connection ) 💡